src/ebonite/ext/flask/server.py

```diff
@@ -84,6 +86,12 @@ class FlaskServer(BaseHTTPServer):
         current_module_path('build_templates', 'uwsgi.ini')
     ]
 
+    # TODO somehow make this depend on builder implementation?
```

Comment:
As I still couldn't imagine non-Docker builder in our system for the nearest decade I propose to put a note here (not to forget about this in future) and leave it as it is.